src/Lykke.Sdk/LykkeServiceCollectionContainerBuilderExtensions.cs

```diff
@@ -83,40 +102,18 @@ public static class LykkeServiceCollectionContainerBuilderExtensions
                 builder.RegisterInstance(settings.Nested(x => x.MonitoringServiceClient));            
 
             builder.RegisterInstance(serviceOptions);
+            builder.RegisterType<AppLifetimeHandler>().AsSelf().SingleInstance();
 
-            var logger = LoggerFactory.CreateLogWithSlack(
-                builder, 
-                serviceOptions.Logs.TableName, 
-                settings.ConnectionString(serviceOptions.Logs.ConnectionString), 
-                settings.CurrentValue.SlackNotifications
-            );
-
-            builder.RegisterInstance(logger);
-            builder.RegisterAssemblyModules(settings, logger, Assembly.GetEntryAssembly());
+            builder.RegisterAssemblyModules(settings, Assembly.GetEntryAssembly());
```

Comment:
This should be place after builder.Populate(services);
Reason:
"if there is a redis cache registration in the module, then calling .Populate(services) after the modules registraions will reset the cache to in mem implementation"

Reply:
Rebased to master - fixed